features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2762,4 +2762,14 @@
         	<Cookie name="{{cookie}}"/>
         	{% endfor %}
     </CookiesToInvalidate>
+
+    <!-- Claim to store localization code  -->
+    <!--    By setting true localization code will be stored inside the http://wso2.org/claims/locality-->
+    <!--    By setting false localization code will be stored inside the http://wso2.org/claims/local-->
+    {% if claims is defined && claims.use_legacy_localization_claim is defined %}
```

Comment:
formatting issue.
lines 2770 to 2773 should be aligned with 2769

Reply:
Fixed in commit 742fb3e45ddfa401035bfc2397f3aebf28b8cc20